Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM python:3.8-slim as base
 
-FROM base as builder
 # Any python libraries that require system libraries to be installed will likely
 # need the following packages in order to build
-RUN apt-get update && apt-get install -y build-essential git
+RUN apt-get update && \
```

Comment:
this should be in the base stage, since it doesn't need to be re-executed each time and can be cached